packages/get-starknet/package.json

```diff
@@ -18,7 +18,7 @@
     "prettier": "prettier --write \"src/**/*.ts\"",
     "lint": "eslint 'src/*.{js,ts,tsx}' --max-warnings 0 -f json -o eslint-report.json",
     "lint:fix": "eslint '**/*.{js,ts,tsx}' --fix",
-    "test:unit": ""
+    "test": "jest --passWithNoTests"
```

Comment:
Maybe we can add a constraint here ?

Reply:
remarks: Add coverage when we have full test implemented